src/ProjectTemplates/Web.ProjectTemplates/Worker-FSharp.fsproj.in

```diff
@@ -0,0 +1,19 @@
+<Project Sdk="Microsoft.NET.Sdk.Worker">
+
+  <PropertyGroup>
+    <TargetFramework>${DefaultNetCoreTargetFramework}</TargetFramework>
+    <UserSecretsId>dotnet-Company.Application1-53bc9b9d-9d6a-45d4-8429-2a2761773502</UserSecretsId>
```

Comment:
I just copied this GUID from the C# template, but if we need a new one I'll get a hot one from hotguids: http://www.secretgeek.net/hotGuids/index.htm

Reply:
As a best practice, we use https://twitter.com/guidoftheday

Reply:
Ew, these are so old though. I need hot guids fresh from the oven